RaveBusinessLogic/VBET.xml

```diff
@@ -113,8 +113,8 @@
           </Repeater>
           <Node label="Combined Evidence Rasters">
             <Children>
-              <Node label="Channel Evidence" xpath="Intermediates/Raster[@id='EVIDENCE_CHANNEL']" type="raster" symbology="" transparency="40" />
-              <Node label="Topographic Evidence" xpath="Intermediates/Raster[@id='EVIDENCE_TOPO']" type="raster" symbology="" transparency="40" />
+              <Node label="Channel Evidence" xpath="Intermediates/Raster[@id='EVIDENCE_CHANNEL']" type="raster" symbology="channelevidence" transparency="40" />
+              <Node label="Topographic Evidence" xpath="Intermediates/Raster[@id='EVIDENCE_TOPO']" type="raster" symbology="topoevidence" transparency="40" />
             </Children>

```

Comment:
Does this collide with or conflict with other vbet_symbology business logic pull request (#118)  I just approved? Please avoid putting pull requests in simultaneously on different branches that make changes to same files.